api/src/main/java/org/openmrs/api/ProviderService.java

```diff
@@ -77,6 +78,13 @@
 	@Authorized( { PrivilegeConstants.PURGE_PROVIDERS })
 	public void purgeProvider(Provider provider);
 	
+	/**
+	 * Create a given Provider 
+	 * @param providers to be created
+	 */
+	@Authorized({PrivilegeConstants.CREATE_PROVIDERS})
```

Comment:
Is this simply saving a provider?

Reply:
Sure this creates and saves a provider given User privilege

Reply:
So why do we need this function if we already have `saveProvider()`?

Reply:
This function saves a provider of a given id